save-backend/src/main/kotlin/org/cqfn/save/backend/service/LnkUserProjectService.kt

```diff
@@ -29,4 +31,16 @@ class LnkUserProjectService(private val lnkUserProjectRepository: LnkUserProject
         .ifEmpty { listOf(Role.VIEWER) }
         .singleOrNull()
         ?: throw IllegalStateException("Multiple roles are set for userId=$userId and project=$project")
+
+    /**
+     * Set role of [user] on a project [project] to [role]
+     */
+    @Suppress("KDOC_WITHOUT_PARAM_TAG")
+    fun addRole(user: User, project: Project, role: Role) {
```

Comment:
May be not `addRole` but something like `save`? We don't check of existence of `user/project` and so on here, and just saving